### ISSUE TITLE
Being able to turn off embedded/internal controller with a click of a button

### ISSUE THREAD

**Issue by philipjholm** [Valkirie/HandheldCompanion#720]
Would it be possible to add a button to the overlay menu to quickly being able to turn off or on the embedded controller? This is needed when wanting to play from the couch when connected to TV and not using the handheld itself for controlling. 

I really would wish for this to be a thing I could do with my ROG Ally 👍

**Comment by CasperH2O** [comment]
@philipjholm on my Ally, when docked, on the controller page, I "connect" the external controller and done. You'd still have to do this manually after docking though, which could be improved.

**Comment by philipjholm** [comment]
Yeah making it automatic when external monitor is connected would make this a very seamless experience and would improve the  docking process!

**Comment by CasperH2O** [comment]
Related to #116
